ui/src/app/pages/admin/workspace/basic-information.tsx

```diff
@@ -24,6 +24,9 @@ export const BasicInformation = ({ workspace, activeStatus }: Props) => (
       <WorkspaceInfoField labelText='Workspace Name'>
         {workspace.name}
       </WorkspaceInfoField>
+      <WorkspaceInfoField labelText='Terra Name (often incorrectly called "id")'>
+        {workspace.id}
```

Comment:
changing this "id" to "terraName" will be a big effort - well outside the scope of this PR